glide.yaml

```diff
@@ -29,7 +29,7 @@ import:
   - tools/cache
   - tools/clientcmd
 - package: github.com/projectcalico/libcalico-go
-  version: 4db56aa2f763478265b424ef04988cc6ddd30537
+  version: master
```

Comment:
having this to master can cause problems, since glide will use whatever is in the cache for "master" it won't actually check if master has been updated on github. Better to remove this line. (was bitten by this earlier)